lms/views/lti/config.py

```diff
@@ -14,3 +14,63 @@ def config_xml(request):
         "launch_url": request.route_url("lti_launches"),
         "content_item_url": request.route_url("content_item_selection"),
     }
+
+
+@view_config(
+    route_name="lti.v13.canvas.config",
```

Comment:
This is canvas only for now. 

We could try to do a generic view and add to that the canvas specific confic. We don't know exactly what will be different in other LMSs so I rather wait until we need it and keep this is simple for now.